sandbox/lua/modules/graphite.lua

```diff
@@ -0,0 +1,135 @@
+-- This Source Code Form is subject to the terms of the Mozilla Public
+-- License, v. 2.0. If a copy of the MPL was not distributed with this
+-- file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+--[[
+Module contains utilities for Graphite
+--]]
+-- Imports
+require "string"
+require "math"
+require "table"
+
+local M = {}
+
```

Comment:
Yeah, i'm relatively new to lua and alert.lua was my entrypoint. I've spotted 3 different conventions for modules and i wasn't sure which standard here. ;-) Can i start working on #1496 after i'll clean up this PR?